### ISSUE TITLE
requirements

### ISSUE THREAD

**Issue by PawanR-deveops** [PawanR-deveops/pawanR#1]
absl-py==2.0.0
altair==5.1.2
astunparse==1.6.3
attrs==23.1.0
blinker==1.7.0
cachetools==5.3.2
certifi==2023.7.22
charset-normalizer==3.3.2
click==8.1.7
colorama==0.4.6
contourpy==1.2.0
cycler==0.12.1
distlib==0.3.7
filelock==3.13.1
flatbuffers==23.5.26
fonttools==4.44.0
gast==0.5.4
gitdb==4.0.11
GitPython==3.1.40
google-auth==2.23.4
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.59.2
h5py==3.10.0
idna==3.4
importlib-metadata==6.8.0
Jinja2==3.1.2
joblib==1.3.2
jsonschema==4.19.2
jsonschema-specifications==2023.7.1
keras==2.14.0
kiwisolver==1.4.5
libclang==16.0.6
Markdown==3.5.1
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.8.1
mdurl==0.1.2
ml-dtypes==0.2.0
numpy==1.26.1
oauthlib==3.2.2
opencv-python==4.8.1.78
opt-einsum==3.3.0
packaging==23.2
pandas==2.1.2
Pillow==10.1.0
platformdirs==3.11.0
protobuf==4.25.0
pyarrow==14.0.0
pyasn1==0.5.0
pyasn1-modules==0.3.0
pydeck==0.8.1b0
Pygments==2.16.1
pyparsing==3.1.1
python-dateutil==2.8.2
pytz==2023.3.post1
referencing==0.30.2
requests==2.31.0
requests-oauthlib==1.3.1
rich==13.6.0
rpds-py==0.12.0
rsa==4.9
scikit-learn==1.3.2
scipy==1.11.3
six==1.16.0
smmap==5.0.1
streamlit==1.28.1
tenacity==8.2.3
tensorboard==2.14.1
tensorboard-data-server==0.7.2
tensorflow==2.14.0
tensorflow-estimator==2.14.0
tensorflow-intel==2.14.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==2.3.0
threadpoolctl==3.2.0
toml==0.10.2
toolz==0.12.0
tornado==6.3.3
typing_extensions==4.8.0
tzdata==2023.3
tzlocal==5.2
urllib3==2.0.7
validators==0.22.0
virtualenv==20.24.6
watchdog==3.0.0
Werkzeug==3.0.1
wrapt==1.14.1
zipp==3.17.0
